navigation/design.md

```diff
@@ -0,0 +1,35 @@
+# Navigation
+
+## Scope Selector
+![scope selector](img/scope-selector.png)
+- The scope selector is a dropdown just below the masthead the enables users to focus their work on a specific namespace. In the future, this selector could be used to select from a number of logical groups created by the user.
+
+
+![open scope selector](img/scope-selector-open.png)
+- When a single object is selected, the scope selector changes to show the namespace of that object. Navigating back to non-namespace-specific page such as a list or graph view page changes the selector back to `all namespaces` unless the scope selector was manually set to a specific namespace previously.
+
+
+## Breadcrumbs
+![breadcrumbs](img/navigation-breadcrumbs-leaf.png)
+- Breadcrumbs link the current page to its parent in order to show the relationships between objects and provide a way to navigate to intermediate levels of hierarchy that aren't easily reachable from the main navigation.
```

Comment:
In the screenshot, there is 'productpage - bookinfo' in the breadcrumb and 2 arrows next to it. The 'bookinfo' is the namespace, which is also selected on the upper toolbar. Why is it also part of the breadcrumb, and to the right of the service named productpage?